packages/gallery/src/components/helpers/dimensionsHelper.js

```diff
@@ -74,42 +63,14 @@ class DimensionsHelper {
   }
 
   isUnknownWidth(container = this.container) {
-      //if the container width is not a number, it is unknownWidth (e.g.: "", "100%", "calc(100% + -160px)")
-      return !(container.width > 0);
+    //if the container width is not a number, it is unknownWidth (e.g.: "", "100%", "calc(100% + -160px)")
+    return !(container.width > 0);
   }
 
   isUnknownHeight(container = this.container) {
     //if the container height is not a number, it is unknownHeight (e.g.: "", "100%", "calc(100% + -160px)")
     return !(container.height > 0);
   }
-
-  tryCalcAndSetContainerWidth() {
-    const boundingRect = this.calcBoundingRect();
-    const calcWidth = boundingRect && boundingRect.width;
-    if (calcWidth) {
-      this.container.width = calcWidth;
-      if (utils.isVerbose()) {
-        console.log('Pro-Gallery calculated width');
-      }
-    }
-  }
-
-  tryCalcAndSetContainerHeight() {
-    const boundingRect = this.calcBoundingRect();
-    const calcHeight = boundingRect && boundingRect.height;
-    if (calcHeight > 0) {
-      this.container.height = calcHeight;
-      if (utils.isVerbose()) {
-        console.log('Pro-Gallery calculated height');
-      }
-    } else if (calcHeight === 0) {
-      this.container.height = 200; //default height, just to not be 0
-      if (utils.isVerbose()) {
-        console.log('Pro-Gallery calculated height of 0, will set manually to 200');
-      }
-    }
-  }
-
   getGalleryWidth() {
```

Comment:
You can remove the `domWidth()`
So that the `getGalleryWidth` only does the dimension fix